src/Native/Runtime/windows/CoffNativeCodeManager.cpp

```diff
@@ -86,6 +117,14 @@ static PTR_VOID GetUnwindDataBlob(TADDR moduleBase, PTR_RUNTIME_FUNCTION pRuntim
 
     return pUnwindInfo;
 
+#elif defined(_TARGET_X86_)
+
+    PTR_UNWIND_INFO pUnwindInfo(dac_cast<PTR_UNWIND_INFO>(moduleBase + pRuntimeFunction->UnwindInfoAddress));
+
+    *pSize = sizeof(UNWIND_INFO);
```

Comment:
It may be better to add:

```
PORTABILITY_ASSERT("GetUnwindDataBlob");
*pSize = 0;
return NULL;
```

 to the `#else` part of the ifdef - instead of the `#error unexpected target architecture`.